src/components/Sheet/README.md

```diff
@@ -0,0 +1,17 @@
+## MobileModal
+
+Sheet component for mobile devices
+
+### PropTypes
+
+| Property                 | Type       | Required | Default     | Description                                                                                                                                                                 |
+| :----------------------- | :--------- | :------: | :---------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| visible                  | `boolean`  |    ✓     |             | Show/hide sheet                                                                                                                                                             |
```

Comment:
Why it should be a required property?

Reply:
It was required in the original component. Maybe to highlight that you should to specify `visible` property to control visibility of the component, it's useless without `visible` property.